syntax = "proto3";
package rdma;

message GetNodeIDForIpPortRequest {
    string ipPort = 1;
}

